types/ckeditor__ckeditor5-alignment/src/utils.d.ts

```diff
@@ -1,8 +1,24 @@
 import { Locale } from "@ckeditor/ckeditor5-utils";
 import { AlignmentFormat } from "./alignmentediting";
-
-export const supportedOptions: ["left", "right", "center", "justify"];
-
-export function isDefault(alignment: string, locale: Locale): boolean;
-export function isSupported(option: string): boolean;
-export function normalizeAlignmentOptions(configuredOptions: Array<string | AlignmentFormat>): AlignmentFormat[];
+/**
+ * The list of supported alignment options:
+ *
+ * * `'left'`,
+ * * `'right'`,
+ * * `'center'`,
+ * * `'justify'`
+ */
+export const supportedOptions: readonly ["left", "right", "center", "justify"];
+/**
+ * Checks whether the passed option is supported by {@link module:alignment/alignmentediting~AlignmentEditing}.
+ */
+export function isSupported(option: any): boolean;
+/**
+ * Checks whether alignment is the default one considering the direction
+ * of the editor content.
+ */
+export function isDefault(alignment: AlignmentFormat['name'] | undefined, locale: Locale): boolean;
```

Comment:
`execute({value: undefined})` is valid, so this function would receive `undefined`.